content_scripts/mode_normal.js

```diff
@@ -213,6 +213,22 @@ const NormalModeCommands = {
     }
   },
 
+  findSelected() {
+    let selection = window.getSelection().toString();
+    if (!selection) return;
+    FindMode.updateQuery(selection);
+    FindMode.saveQuery();
+    return FindMode.findNext(false);
+  },
+
+  findSelectedBackwards() {
+    let selection = window.getSelection().toString();
+    if (!selection) return;
+    FindMode.updateQuery(selection);
+    FindMode.saveQuery();
+    return FindMode.findNext(true);
```

Comment:
Since both the forward and backward implementation use the same code except for the forward/backward boolean, it might make sense to use a helper function that accepts a boolean. It just depends what @philc prefers for maintainability.